perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/PerunBlImpl.java

```diff
@@ -407,6 +408,11 @@ public void setSearcherBl(SearcherBl searcherBl) {
 		this.searcherBl = searcherBl;
 	}
 
+	@Override
+	public boolean isThisPerunReadOnly() {
```

Comment:
In my opinion is better to have this method also in object Perun for easier usage.